### PR TITLE
ISPN-16069 Allow the indexes to start with embedded setup

### PR DESCRIPTION
**Pull request by fax4ever**
https://issues.redhat.com/browse/ISPN-16069

### CODE REVIEW

**Comment by fax4ever** [comment]
Rebased!

**Review by pruivo — COMMENTED** [review]
LGTM, just some suggestions :+1:

File: documentation/src/main/asciidoc/topics/ref_indexing_configuration.adoc [hunk 1]
```diff
@@ -81,6 +81,20 @@ Set the `indexing-mode` to `manual`:
 include::xml/indexing_manual.xml[]
 ----
 
+[discrete]
+== Use
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/ref_indexing_configuration.adoc:85]: `Use Java Entities` maybe?

Comment (fax4ever) [reply documentation/src/main/asciidoc/topics/ref_indexing_configuration.adoc]: Good idea. I applied it.

**Comment by pruivo** [comment]
merged! thanks @fax4ever !

**Comment by fax4ever** [comment]
Thanks Pedro!

**Pull request merged**